server/service/transaction_service.rs

```diff
@@ -1342,6 +1342,9 @@ impl TransactionService {
             let named_outputs = pipeline.rows_positions().unwrap();
             let descriptor: StreamQueryOutputDescriptor = named_outputs.clone().into_iter().sorted().collect();
             let initial_response = StreamQueryResponse::init_ok_rows(&descriptor, Read);
+            #[cfg(debug_assertions)]
+            dump_query_structure(pipeline.query_structure());
```

Comment:
This should be on `trace!` + the debug flag you have if you want to check it into master IMO